packages/opentelemetry-api/src/api/global-utils.ts

```diff
@@ -38,7 +38,11 @@ type MyGlobals = Partial<{
   [GLOBAL_TRACE_API_KEY]: Get<TracerProvider>;
 }>;
 
-export const _global = global as typeof global & MyGlobals;
+export const _global = (typeof globalThis === 'object'
```

Comment:
I think we should define `global` per platform and then use it here.
See example how it is done for example here
https://github.com/open-telemetry/opentelemetry-js/tree/b1255217a6f491756335fcb0945f4c8f205e0200/packages/opentelemetry-core/src/platform
You will need to define also a `browser` section in `package.json`.

Reply:
Yes, but this case is not identical to that one. Ideally, `globalThis` is preferred for its cross-platform design. However, `globalThis` only available after Node.js v12 and very new versions of browsers. 

For those platform-dependent setups, I do believe they should go into this pattern. But for _the global_ thing, I believe this is ok-ish (or `globalThis` would be referenced both in their `platform/node/global.ts` and `platform/browser/global.ts`).

Reply:
You have already added a `window` which doesn't exist in node so this is wrong approach anyway and for such purposes the `platform` has been done.

Reply:
and IE still doesn't support globalThis

Reply:
As the person who caused this bug in the first place, think I'll defer to @obecny's web browser expertise in this regard. Generally, I don't think we should take advantage of features which are very new and only work on a small subset of very up-to-date installations.